homeassistant/components/camera/arlo.py

```diff
@@ -158,15 +161,48 @@ def disable_motion_detection(self):
         self._motion_status = False
         self.set_base_station_mode(ARLO_MODE_DISARMED)
 
+    @staticmethod
+    def clean_attr(attr):
+        """Return unknown if attribute is None (non-subscriptable)."""
+        return str(attr) if attr is not None else STATE_UNKNOWN
+
     def update(self):
         """Add an attribute-update task to the executor pool."""
-        self.attrs[ATTR_BATTERY_LEVEL] = self._camera.get_battery_level
-        self.attrs[ATTR_BRIGHTNESS] = self._camera.get_battery_level
-        self.attrs[ATTR_FLIPPED] = self._camera.get_flip_state,
-        self.attrs[ATTR_MIRRORED] = self._camera.get_mirror_state,
-        self.attrs[
-            ATTR_MOTION] = self._camera.get_motion_detection_sensitivity,
+        # pylint: disable=W0212
+        base_stations = self._camera._session.base_stations
```

Comment:
@jwillaz  https://pypi.python.org/pypi/pyarlo/0.0.9 has been published.  You can do something like:
```python
base_station = self._camera.base_station
if base_station:
    base_station._refresh_rate = SCAN_INTERVAL.total_seconds()
```